internal/integration/http/http.go

```diff
@@ -28,6 +29,7 @@ type Config struct {
 	Headers          map[string]string `json:"headers"`
 	EventEndpointURL string            `json:"eventEndpointURL"`
 	Marshaler        string            `json:"marshaler"`
+	Timeout          int64             `json:"timeout"` // Milliseconds
```

Comment:
I think it would be better to directly use a `time.Duration` type here. That way, it is not needed to do any type conversions. A `time.Duration` works with JSON encoding / decoding and will be automatically encoded as string (e.g. "10s").

Reply:
I've made this change in b0e982ac29369c058d19f6558acfe7781b8078f7.